### DISCUSSION TITLE
Account aliases

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#11071] [Other Feature Feedback, Questions, & Ideas]
If you have multiple git user.name and user.email, e. g. private and company, it could be useful to add aliases for your GitHub account.

Example:
You commit and push something with your company git account it should be identified with your GitHub account which you use for private stuff and vice versa

**Comment by siddhpant** [comment]
You can add the email to your account from settings. You can have multiple emails linked in an account.

GitHub associates account using emails. `user.name` is not used for identification.

**Comment by llinfeng** [comment]
<img width="1920" height="635" alt="图片" src="https://github.com/user-attachments/assets/67b514f8-6e2a-47a4-8ac9-1d6a034a28e9" />
